### ISSUE TITLE
Allow binary version to be configurable

### ISSUE THREAD

**Issue by weaverryan** [SymfonyCasts/sass-bundle#3]
Currently, we hardcode the version of the binary that we download. Added this as a configuration option probably makes sense. (This is also true for https://github.com/SymfonyCasts/tailwind-bundle)

**Comment by dorxy** [comment]
Hi @weaverryan, I'd be happy to give a go at implementing this, (if satisfactory also for the tailwind bundle).

Do you already have some idea on how you would like to do this? Would you like to have a list of supported versions and check beforehand? or merely failing with a 'you probably have a wrong version configured' message if the binary request in `SassBinary::downloadExecutable` fails?

**Comment by weaverryan** [comment]
> or merely failing with a 'you probably have a wrong version configured' message if the binary request in SassBinary::downloadExecutable fails?

Yup, this :D. If you give us the version, we use that in the URL instead of https://github.com/SymfonyCasts/sass-bundle/blob/main/src/SassBinary.php#L19 and if it fails, it fails. 

So yes please! And on both bundles would be wonderful ❤️

**Comment by bocharsky-bw** [comment]
There's a related PR for TailwindBundle: https://github.com/SymfonyCasts/tailwind-bundle/pull/34
